File: Entities/Concrete/Entities/General/Machine/TramRoll.cs
﻿using Core.Entities.Abstract;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Concrete.Entities.General.Machine
{
    public class TramRoll : IEntity//Tramvay Rulosu
    {
        public Guid Id { get; set; }
        public Guid ParameterId { get; set; }
        public string? RollNumber { get; set; }
        public decimal? RollDiameter { get; set; }
        public string? GroupName { get; set; }
        public string? RollName { get; set; }
        public short? TramCount { get; set; }
        public string? Status { get; set; }
        public string? Location { get; set; }
        public DateTime Optime { get; set; }
        public bool IsActive { get; set; }
    }
}

File: UIPresentation/web/src/app/models/tram-roll/tram-roll.ts
import { LocationEnum } from "src/app/enums/location-enum.enum"
import { RollStatusEnum } from "src/app/enums/roll-status-enum.enum"

export class TramRoll {
  id!:string
  rollNumber!:string
  rollDiameter?:number
  groupName?:string
  rollName?:string
  tramCount?:number
  status!:RollStatusEnum
  location!:LocationEnum
  optime!:Date
  isActive!:boolean
}


